### DISCUSSION TITLE
Home assistant and nodejs

### DISCUSSION THREAD

**Discussion by tfd949** [tagyoureit/nodejs-poolController#826] [Q&A]
We have Home Assistant OS running on our raspberryPi and we are getting ready to install nodeJS but we are confused on how to install it under HA. Please help

Home Assistant 2023.7.2
Supervisor 2023.07.1
Operating System 10.3
Frontend 20230705.1 - latest

Can anyone help us get this up and running.

We have a RPi 3, with a 4 channel power relay, 3 onewire temp sensors, 2 atlas scientific sensors (ph and orp), single stage pump (in one relay), and lights (in another relay)

we would like to incorporate it all into HA if possible.

**Comment by rstrouse** [comment]
Install the @crewski plugin via hacs.  The instructions are in this repo.  https://github.com/Crewski/njsPC-HA

**Reply by tfd949** [reply]
I still need to install nodejs correct?

If so how?

**Reply by rstrouse** [reply]
Here is a quick start guide.  https://github.com/tagyoureit/nodejs-poolController/wiki/Quick-Start-Guide
